linear_scale: 1
max_positive_linear_velocity: 1
max_negative_linear_velocity: -1
